Internship2022TeamPatri/Internship2022TeamPatri/Networking/Services/AuthApiManager.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  AuthApiManager.swift
+//  Internship2022TeamPatri
+//
+//  Created by Vlad Silviu Hagiu on 02.08.2022.
+//
+
+import Foundation
+import FirebaseAuth
+import UIKit
+import FirebaseFirestore
+
+class AuthApiManager {
+    static let sharedInstance = AuthApiManager()
+    private let db = Firestore.firestore()
+    
+    func loginAPI(email: String?, password: String?) {
```

Comment:
You need to pass a completion parameter here so the consumer of this method can handle its result (success or failure) and also you need to have a isLoggedIn flag that can be used outside this manager to know if the user is already loggedIn or not. (For example, on app start, if the user is already logged in you won't show the login screen anymore so you need a Bool flag to know when the user is logged in or not)